citi-onboarding/maracatronics
# Feature/mission values vision

## What i did
- Created the Mission, value and vision model;
- Created the Mission, value and vision controller;
- Created the Mission, value and vision route;

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "feature/mission-values-vision" with ```git checkout feature/mission-values-vision```
3. Now, you need to found the server folder with ```cd server```
4. On the server folder, run the server with ```node index.js``` and go to http://localhost:3001/admin
5. Go to "Missão, valores e visão" model and edit the texts
6. Now, go to the mongoDB account and check if the texts changed
7. On mongoDB account, click in Maraca and, after, in Collections to see the data. 

_The accounts will be in Notion._
